tests/integration_tests/tests/agentless_tests/test_component_type.py

```diff
@@ -100,3 +94,130 @@ def test_component_creation_with_not_existing_blueprint_package(self):
                           self.deploy_application,
                           dsl_path,
                           deployment_id=deployment_id)
+
+
+class ComponentScaleCreation(AgentlessTestCase):
+    component_name = 'component'
+
+    def test_given_deployment_name_with_auto_suffix_inc_option(self):
+        basic_blueprint_path = resource('dsl/basic.yaml')
+        self.client.blueprints.upload(basic_blueprint_path,
+                                      entity_id='basic')
+        deployment_id = 'd{0}'.format(uuid.uuid4())
+        main_blueprint = """
```

Comment:
This should be placed under `tests/integration_tests/resources/dsl` as a blueprint yaml file with an appropriate name

Reply:
It's not efficient (space or run time) to create a blueprint file for each test, and I think it's more readable when it's close together